ros2test/ros2test/command/test.py

```diff
@@ -24,9 +28,24 @@ class TestCommand(CommandExtension):
     def add_arguments(self, parser, cli_name):
         """Add arguments to argparse."""
         launch_testing.launch_test.add_arguments(parser)
+        parser.add_argument(
+            '--disable-isolation', action='store_true', default=False,
+            help='Disable automatic ROS_DOMAIN_ID isolation.'
+            'If ROS_DOMAIN_ID is already set, launch_test will respect and use it. If it is not'
+            ' set, a ROS_DOMAIN_ID not being used by another ros2 test will be picked. '
```

Comment:
@ivanpauno consider `used` instead of `picked`.

Reply:
`...used by another ros2 test will be used.`
I don't like repeating `used`. Maybe `chosen`?